crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -828,6 +828,14 @@ impl C8YHttpProxy for FakeC8YHttpProxy {
     ) -> Result<String, SMCumulocityMapperError> {
         Ok("123".into())
     }
+
+    async fn upload_config_file(
```

Comment:
I feel it's wrong that we add a test for config inside of `tedge_mapper` crate. The config plugin doesn't use mapper at all.